woo-razorpay.php

```diff
@@ -82,9 +82,7 @@ function() {
 
 function woocommerce_razorpay_init()
 {
-    add_action("woocommerce_update_options_advanced", 'hposInstrumentation');
-
-    function hposInstrumentation()
+    add_action("woocommerce_update_options_advanced", function()
```

Comment:
@rohitcbr has this been tested for any event?

Reply:
@rohitcbr , why this change required?